README.md

```diff
@@ -1,16 +1,16 @@
 # opentracing go-redis
 
-[OpenTracing](http://opentracing.io/) instrumentation for [go-redis](https://github.com/go-redis/redis).
+[OpenTracing](http://opentracing.io/) instrumentation for [go-redis](https://github.com/go-redis/redis) with support of `process` and `processPipeline` commands.
 
 ## Install
 
 ```
-go get -u github.com/smacker/opentracing-go-redis
+go get -u github.com/ticketmaster/opentracing-go-redis
 ```
 
 ## Usage
 
-Clone redis client `c := otredis.WrapRedisClient(ctx, c)` with a span.
+Clone redis client `c := otredis.WrapWithOpenTracing(ctx, c)` with a span.
```

Comment:
Not needed anymore as the same name is used

Reply:
:+1: